manifests/pe_metric.pp

```diff
@@ -38,10 +40,44 @@
   }
 
   $script_file_name = "${scripts_dir}/${metric_script_file}"
+  $conversion_script_file_name = "${scripts_dir}/json2timeseriesdb"
+
+  if empty($override_metrics_command){
+    $metrics_base_command = "${script_file_name} --metrics_type ${metrics_type} --output-dir ${metrics_output_dir}"
+
+    if !empty($metrics_server_info) {
+      $metrics_server_hostname = $metrics_server_info['hostname']
+      $metrics_server_port     = $metrics_server_info['port']
+      $metrics_server_type     = $metrics_server_info['metrics_server_type']
+      $metrics_server_db       = $metrics_server_info['db_name']
+
+      if empty($metrics_server_db) and $metrics_server_type == 'influxdb'  {
+        fail( 'When using an influxdb server you must provide the db_name to store metrics in' )
+      }
+
+      $local_metrics_command = "${metrics_base_command} | ${conversion_script_file_name} --netcat ${metrics_server_hostname} --convert-to ${metrics_server_type}"
+
+      $port_metrics_command = empty($metrics_server_port) ? {
+        false => "${local_metrics_command} --port ${metrics_server_port}",
+        true  => $local_metrics_command,
+      }
+
+      $metrics_command = $metrics_server_type ? {
+        'influxdb' => "${port_metrics_command} --influx-db ${metrics_server_db} > /dev/null",
+        'graphite' => "${port_metrics_command} > /dev/null",
+        'splunk_hec' => "${port_metrics_command} | puppet splunk_hec --sourcetype puppet:summary --saved_report > /dev/null",
```

Comment:
should be the following for splunk_hec (no assumption on $path setting, etc) /opt/puppetlabs/bin/puppet splunk_hec --sourcetype puppet:metrics --pe_metrics